ryanlau/lab1_hard_project/src/hello_world.cpp

```diff
@@ -0,0 +1,6 @@
+#include <iostream>
+#include "../include/hello_world.h"
```

Comment:
please do not include your headers like this, your `include` folder has already been added to your library's include path to include from directly

Reply:
addressed